webapp/views.py

```diff
@@ -399,15 +399,26 @@ def advantage_view():
     new_subscription_start_date = None
     new_subscription_id = None
     open_subscription = flask.request.args.get("subscription", None)
+    is_staging = flask.request.args.get("ua_staging", False)
```

Comment:
You very smartly use the wording "live" and "test" in the env var names, which I like. Because you could switch backend on any environment, so it's more about testing and live than it is about staging and production.

But here you use the word "staging". Maybe this should be "test_backend" or something?